lib/rules/lint-table-groups.js

```diff
@@ -49,31 +45,22 @@ function isAllowedTableChild(node) {
         return ALLOWED_TABLE_CHILDREN.includes(tagNameAttribute.value.chars);
       }
       break;
+    case 'CommentStatement':
+    case 'MustacheCommentStatement':
+      return true;
```

Comment:
Can you add tests for comments?

Reply:
There are tests for comments, but the extra check we had before was causing them to go through a special case. I removed that check entirely, so I don't think we need another test of a comment adjacent to a valid child.

Reply:
You fixed a real bug with ignoring comments so I do think it's worthwhile to add a test to show that bug is fixed :)

Reply:
I generally don't like adding tests that make sure code that has been deleted isn't there, but it was easy enough to add it to an existing test.